app/models/editionable_worldwide_organisation.rb

```diff
@@ -64,6 +65,11 @@ def process_associations_before_save(new_edition)
   include AnalyticsIdentifierPopulator
   self.analytics_prefix = "WO"
 
+  delegate :alternative_format_contact_email, to: :sponsoring_organisation, allow_nil: true
+  def sponsoring_organisation
+    lead_organisations.first
+  end
```

Comment:
Are these required for supporting attachments?

Reply:
Yes, turns out it was also required in this model.

That's because we eventually need it [here](https://github.com/alphagov/whitehall/blob/bf678254b8bdb3930735f569f065c1985b668e1c/app/helpers/govspeak_helper.rb#L31C70-L31C102), which is called from this model's [presenter](https://github.com/alphagov/whitehall/blob/bf678254b8bdb3930735f569f065c1985b668e1c/app/presenters/publishing_api/editionable_worldwide_organisation_presenter.rb#L76C39-L76C63).